### PR TITLE
Add config files for some services on 'qa' box

### PR DESCRIPTION
**Pull request by jimwins**
These reference "secrets" that are kept outside VC in /local/secrets on the box, until we have another strategy for managing secrets.

The qa.php.net config is http-only in anticipation of putting that behind Myra protection and to avoid having to keep track of SSL config here.

### CODE REVIEW

File: qa/apache2/qa.php.net.conf [hunk 1]
```diff
@@ -38,10 +46,16 @@
         Require all denied
     </FilesMatch>
 
-    CustomLog ${APACHE_LOG_DIR}/qa_access.log combined
-    ErrorLog ${APACHE_LOG_DIR}/qa_error.log
+    CustomLog ${APACHE_LOG_DIR}/qa_ssl_access.log combined
+    ErrorLog ${APACHE_LOG_DIR}/qa_ssl_error.log
```
Comment (derickr) [inline qa/apache2/qa.php.net.conf:50]: Do we really need `ssl` in the name?

**Review by derickr — COMMENTED** [review]
I'm not quite sure whether this now reflects what's on the server too? Or whether it is safe to merge this and this then automatically showing up on `qa.php.net` through rsync, because that's what will happen if this is merged.

File: qa/apache2/qa.php.net.conf (hunk 1 above)
Comment (jimwins) [reply qa/apache2/qa.php.net.conf]: No, it was just how it was configured now (and there were two sets of log files for http and https). I've made it more consistent between qa.php.net and news-web.php.net now.

**Comment by jimwins** [comment]
> I'm not quite sure whether this now reflects what's on the server too?

Yes, it's all consistent with how these services are configured now. Once this is merged, I'll actually move the old files out of the way and symlink into `/local/system` for these, and switch the crontab setup to use `cron-box`.

**Comment by derickr** [comment]
OK - would you mind adding a script under `qa.php.net` to make these symlinks / copying the cron-box file automatically perhaps? That way, that is also reproducible.

Other than that, this looks good to me, but I'm still a little apprehensive of merging this :-).

**Comment by jimwins** [comment]
> OK - would you mind adding a script under `qa.php.net` to make these symlinks / copying the cron-box file automatically perhaps? That way, that is also reproducible.

Sure, I added qa/install-config.sh. It backs up the existing files (and crontab) to a directory in /tmp and then loads the new crontab and links the new config files into place. Only the backup part has been tested.

> Other than that, this looks good to me, but I'm still a little apprehensive of merging this :-).

Should be easy to fix any problems that crop up, the config isn't all that complicated.

**Comment by derickr** [comment]
I've merged this, but I am not sure how wise it is to have these READMEs for servers in here, as it is bound to get out of date really quickly.

**Pull request merged**